lib/src/style/interactive_features/interactive_features.dart

```diff
@@ -0,0 +1,44 @@
+part of mapbox_maps_flutter;
+
+// A single tap interaction.
+class TapInteraction extends Interaction {
+  TapInteraction(FeaturesetDescriptor featuresetDescriptor,
+      {String? filter, double? radius, bool stopPropagation = true})
+      : super(
+            featuresetDescriptor: featuresetDescriptor,
+            interactionType: InteractionType.TAP,
+            filter: filter,
+            radius: radius,
+            stopPropagation: stopPropagation);
+}
+
+// A long tap interaction
+class LongTapInteraction extends Interaction {
+  LongTapInteraction(FeaturesetDescriptor featuresetDescriptor,
+      {String? filter, double? radius, bool stopPropagation = true})
+      : super(
+            featuresetDescriptor: featuresetDescriptor,
+            interactionType: InteractionType.LONG_TAP,
+            filter: filter,
+            radius: radius,
+            stopPropagation: stopPropagation);
+}
+
+extension Featureset on FeaturesetDescriptor {
+  // A Featureset of buildings in the Standard style
+  static FeaturesetDescriptor standardBuildings({String importId = "basemap"}) {
```

Comment:
Can we take a similar approach to Android/iOS and generate this?

Reply:
I'd like to, but didn't want to expand the scope of this PR further. I can create a ticket.